views/includes/inputs/address/service-address-input.html

```diff
@@ -63,7 +63,7 @@
     value: service_address_county
   }) }}
 
-  {% set countryField = service_address_country %}
+  {% set countryField = service_address_country if service_address_country %}
```

Comment:
We do not need this condition

Reply:
it's weird, it seems to fix the problem, i'll see if there's another way